packages/coinjoin/src/client/round/transactionSigning.ts

```diff
@@ -129,11 +129,14 @@ export const transactionSigning = async (
                 );
                 throw new Error('Wittiness not provided');
             }
+
+            round.setSessionPhase(SessionPhase.AwaitingCoinjoinTransaction);
```

Comment:
AwaitingSignature ?

Reply:
it's right before `getTransactionData`, before signing, no? Step 4.1 [in notion](https://www.notion.so/satoshilabs/CoinJoin-states-and-phases-ab33f8d65cc5483f922e455ef29c24ff)